### PR TITLE
fix func of hex and type of blob

### PR DESCRIPTION
**Pull request by iamlinjunhong**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5346 

## What this PR does / why we need it:

### CODE REVIEW

**Review by ouyuanning — APPROVED** [review]

File: pkg/container/types/types.go [hunk 1]
```diff
@@ -81,6 +81,12 @@ const (
 	RowidSize = 16
 )
 
+const (
+	CharNormal int32 = 0
```
Comment (nnsgmsone) [inline pkg/container/types/types.go:85]: Is hex a type?

Comment (iamlinjunhong) [reply pkg/container/types/types.go:85]: Yes, blob is implemented by varchar, so it needs to be distinguished

Comment (nnsgmsone) [reply pkg/container/types/types.go:85]: Are hex and blob the same type?

Comment (nnsgmsone) [reply pkg/container/types/types.go:85]: Isn't a blob a text with a binary character set.

Comment (nnsgmsone) [reply pkg/container/types/types.go:85]: Isn't a blob just a separate type. Isn't T_blob?

Comment (iamlinjunhong) [reply pkg/container/types/types.go:85]: blob can store data of hex, for example:
create table t1 (a blob);
insert into t1 values(0x01020304FFFFFFFF0000000CAAC0000);
select * from t1;
a
0x01020304FFFFFFFF0000000CAAC0000

**Review by aunjgr — APPROVED** [review]

File: pkg/container/types/types.go (hunk 1 above)
Comment (sukki37) [reply pkg/container/types/types.go:85]: It seems not correct.
image

Comment (iamlinjunhong) [reply pkg/container/types/types.go:85]: The type of vector a is varchar which should be blob when read table t1. I am confused by the people who implement this type. The result of show create from t1 is that the type of a is text. The person who implements blob needs to organize.

Comment (iamlinjunhong) [reply pkg/container/types/types.go:85]: This pr will be closed. This solution will indeed cause some problems

**Pull request closed without merging**